### DISCUSSION TITLE
Limited scope GitLab sign on

### DISCUSSION THREAD

**Discussion by kryptec** [200ok-ch/organice#821] [Q&A] [answered]
Hi, 

I was wondering if I could add the app to my home screen from organice.200ok.ch, and then authorize access to GitLab with a limited scope. 

I would prefer to grant it access to a single repo rather than my whole account.

Thanks for the help!

**Comment by munen** [comment, marked as answer]
Hi Elliot

Thank you for using organice 🙏

I am not aware that GitLab supports Repo scoped authorizations. But if it does, you are welcome to provide a PR which adds such an option to the `/sign_in` page. Would you be interested to work on such a PR? I'm happy to do a code review and support you with testing.

**Reply by kryptec** [reply]
Thanks for your reply!

I'll look into it, but I suspect it's somewhat beyond my expertise to code it.

**Reply by munen** [reply]
Godspeed🤞

**Comment by kryptec** [comment]
It looks like you can scope API permissions to specific projects on GitLab, but not on the free tier. So I'm going to put a pin in this idea for now.

**Reply by munen** [reply]
Thank you for researching it :pray:  

If you still remember the link to the documentation, you could leave it here for future investigation. Maybe GitLab changes it at some point. Or even if not, it's good for the next person with the same question^^
